Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -80,12 +92,21 @@ @implementation FSTSpecTests
 }
 
 - (BOOL)shouldRunWithTags:(NSArray<NSString *> *)tags {
-  @throw FSTAbstractMethodException();  // NOLINT
+  if ([tags indexOfObject:kNoIOSTag] != NSNotFound) {
+    return NO;
+  } else if ([tags indexOfObject:kMultiClientTag] != NSNotFound) {
+    return NO;
+  } else if (!kRunBenchmarkTests && [tags indexOfObject:kBenchmarkTag] != NSNotFound) {
+    return NO;
+  }
+  return YES;
 }
 
 - (void)setUpForSpecWithConfig:(NSDictionary *)config {
   // Store persistence / GCEnabled so we can re-use it in doRestart.
   NSNumber *GCEnabled = config[@"useGarbageCollection"];
+  NSNumber *numClients = config[@"numClients"];
```

Comment:
The android port allows this parameter ('numClients') to be absent. Whereas here (I think) it must be present, or (I assume) XCTAssertEqualObjects will fail.

You solve this down below by allowing the object to be the correct value or nil, but at the expense of a less good assert. (AssertTrue vs AssertEqual).

In android, you get to give it a default value which solves the problem nicely. Can you do something similar here? It's an nsdictionary, which I don't think has this concept, but maybe:

```
[config objectForkey:@"numClients"] ?: @1
```

or
```
NSNumber *numClients = config[@"numClients"];
if (numClients == nil) numClients = @1;  // or whatever the syntax is for this. My obj-c foo is weak.
```

If you go the second route, a helper function may be useful.
```
NSNumber *numClients = objectFromDictionary:config key:@"numClients" default:@1
```

Optional. If you decide to do either of these, they may be better as a followup, since I think they'd impact most of this file and be otherwise unrelated to this change.

Reply:
The field is not optional and so should always be said. That said, since this is JSON and we don't use the value, we might as well ignore it. I have move the assert into its own if-statement.